app/models/cart.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class Cart
+  attr_accessor :items
+
+  def initialize(params = { 'items' => {} })
+    params = { 'items' => {} } if params.nil?
```

Comment:
use new syntax for hash styling and use symbols instead of strings https://gitlab.sumatosoft.com/employees/ruby-style-guide/-/tree/master#symbols-as-keys

Reply:
I have that problem and I am not sure that use active_record_store is right decision
https://stackoverflow.com/questions/25151224/why-rails-4-1-cookie-session-serializer-converts-nested-session-keys-from-symbol